### PR TITLE
Kisilov_Vadim

### PR DESCRIPTION
**Pull request by Kisilov-Vadim**

### CODE REVIEW

File: src/countOccurrences.js [hunk 1]
```diff
@@ -18,7 +18,13 @@
...
  */
 function countOccurrences(phrase, letter) {
-  // write code here
+  const reg = new RegExp(letter, 'gi');
+  const coinsidenceArr = phrase.match(reg);
+
+  if (coinsidenceArr == null) {
+    return 0;
+  }
+  return coinsidenceArr.length;
```
Comment (gestap) [inline src/countOccurrences.js]: you can write like this
```suggestion
  return coincidenceArr ? coincidenceArr.length : 0;
```

File: src/countOccurrences.js [hunk 2]
```diff
@@ -18,7 +18,13 @@
  *
  */
 function countOccurrences(phrase, letter) {
-  // write code here
+  const reg = new RegExp(letter, 'gi');
+  const coinsidenceArr = phrase.match(reg);
```
Comment (gestap) [inline src/countOccurrences.js]:
```suggestion
  const coincidenceArr = phrase.match(reg);
```

File: src/countOccurrences.js [hunk 3]
```diff
@@ -18,7 +18,9 @@
...
+  return coinsidenceArr ? coinsidenceArr.length : 0;
```
Comment (gestap) [inline src/countOccurrences.js]: new line
```suggestion

  return coinsidenceArr ? coinsidenceArr.length : 0;
```
also you have mistake `coinSidenceArr` => `coinCidenceArr`

Comment (Kisilov-Vadim) [reply src/countOccurrences.js]: Every time I should add empty line before return? 
Coinsidence - its one word, I should write this word in CamelCase?

Comment (gestap) [reply src/countOccurrences.js]: yes

Comment (gestap) [reply src/countOccurrences.js]: you can look here - https://github.com/mate-academy/style-guides/blob/master/javascript.md#whitespace

**Review by ost-ptk — APPROVED** [review]

File: src/countOccurrences.js (hunk 3 above)
Comment (Kisilov-Vadim) [reply src/countOccurrences.js]: Thank you, I will read this style guide.

**Pull request closed without merging**